### ISSUE TITLE
[Feature request] Live capture to file only

### ISSUE THREAD

**Issue by robgom** [KimiNewt/pyshark#259]
Current LiveCapture interface allows capturing packets to file, but only in addition to capturing them through stdout. That might (or might not) affect performance.
As an alternative, users utilize subprocess to call dumpcap or equivalent by hand. That's tedious and not user friendly, as it requires creating a cmd line string, launching process and waiting for it to complete.
Hence it'd be welcome to provide the functionality through pyshark, which is AFAIK the closest Python package to wireshark.
The functionality would allow specifying that the capture should go straight to file, bypassing stdout altogether. It'd possibly be limited (no packets count, no processing), but will give a convenient way of launching dumpcap (or equivalent).

In short, the motivation for this change would be twofold:
- avoid potential performance issues when using stdout for passing captured packets
- provide convenient abstraction over dumpcap execution

Best regards,
Robert
